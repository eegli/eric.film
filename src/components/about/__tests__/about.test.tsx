import { shallow } from 'enzyme';
import About from '../about.component';

describe('About page', () => {
  it('renders about page', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
    // console.log(wrapper.debug());
  });
});
