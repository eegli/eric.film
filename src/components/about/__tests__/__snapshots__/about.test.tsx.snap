// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page renders about page 1`] = `
<Fragment>
  <aboutstyles__AboutContainer>
    <CardComponent />
    <hr
      style={
        Object {
          "width": "80%",
        }
      }
    />
    <ResumeComponent />
  </aboutstyles__AboutContainer>
</Fragment>
`;
